### PR TITLE
Alternative configuration for GitHub Actions

### PR DESCRIPTION
**Pull request by LukasL28**
Methode 1 is an alternative configuration, where you don't have to work with Github Tokens.

You can use {secrets.GITHUB_TOKEN} to automatically get the token. This is way more user friendly than the current described way.

### CODE REVIEW

File: .github/readme/partials/documentation/setup/action.md [hunk 1]
```diff
@@ -14,6 +14,41 @@ Its `README.md` will be displayed on your user profile:
...
+Rendered metrics will be committed to repository on each run.
+
+![Action update example](/.github/readme/imgs/example_action_update.light.png#gh-light-mode-only)
+![Action update example](/.github/readme/imgs/example_action_update.dark.png#gh-dark-mode-only)
+
+
+- [Continue at 3.1](#3_1)
+
+
+# Methode 2
```
Comment (spenserblack) [inline .github/readme/partials/documentation/setup/action.md:50]:
```suggestion
# Method 2
```

**Review by spenserblack — CHANGES_REQUESTED** [review]
Just some spelling and spacing fixes.

**Comment by spenserblack** [comment]
This brings something else up: perhaps in the section describing how to make a `METRICS_TOKEN`, it should be described why/when that token should be used.

**Comment by LukasL28** [comment]
> This brings something else up: in the section describing how to make a `METRICS_TOKEN`, perhaps it should be described why/when that token should be used.

Isn't it described in 2. ?

**Comment by LukasL28** [comment]
There are Problems with that methode, if you use it for your profile Readme.md.

**Comment by spenserblack** [comment]
>Isn't it described in 2. ?

Just something more explicit, like "You will need to create a custom token if..."

**Comment by lowlighter** [comment]
I think you probably figured out but it's not possible to use `${{ github.token }}`/`${{ secrets.GITHUB_TOKEN }}` for `token` parameter. 

These auto-generated tokens are scoped to the current repository so they're not allowed to access and explore other GitHub data, which is required for all/most plugins. The `committer_token` does work with this though, since it push to said repository ( but this is already the default)

Thanks anyway for trying to improve metrics, I appreciate the effort 😄 !

**Comment by spenserblack** [comment]
I think a lot of users of GitHub Actions are used to `github.token`/`secrets.GITHUB_TOKEN`, and some might assume that it's sufficient (it's good enough for a lot of other actions). It might be a good idea to describe to add something like this:
```
From the Developer settings of your account settings, select Personal access tokens to create a new token.
This token is required to query GitHub API data spanning multiple repositories.
```

**Pull request closed without merging**